src/Controller/FrameworkController.php

```diff
@@ -370,12 +370,14 @@ public function saveTable (Request $request, Session $session){
             $db->CSRFToken = $CSRFToken;
             $db->insert("pl_poste_horaires", $values);
             if (!$db->error) {
-                $session->getFlashbag()->add("success", "Les horaires ont été modifiés avec succès");
+                $msg = "Les horaires ont été modifiés avec succès";
+                $msgType = "success";
             } else {
-                $session->getFlashbag()->add("error", "Une erreur est survenue lors de l'enregistrement des horaires");
+                $msg = "Une erreur est survenue lors de l'enregistrement des horaires";
+                $msgType = "error";
             }
 
-            return $this->redirectToRoute('framework.edit_tab');
+            return $this->redirectToRoute('framework.edit_table', array("id" => $tableauNumero, "cfg-type"=> $post['cfg-type'], "msg" => $msg, "msgType" => $msgType));
```

Comment:
I noticed after most of my corrections that I could not use Flashbags to inform the user, thus I chose to return the message through the URL as legacy code did with a script tag.